### PR TITLE
Small improvements

### PR DESCRIPTION
**Pull request by zdenkoheyvaert**
- Added new features to `help` and `readers` routines (e.g., reader for NU-WRF output);
- Visualization: default values (domain corners, projections, ...) for the plotting functions that are more sensible;
- Improved documentation of the code

### CODE REVIEW

File: help.py [hunk 1]
```diff
@@ -107,11 +107,30 @@ def get_grid_latlon(dc, lat, lon):
...
 
+def latlon_sel(dc, lat, lon, return_indices = False):
+    """
+    Find the closest grid cell of xr.DataArray dc for a given lat and lon in case of a non-orthogonal grid (when the .sel(lat = ..., lon = ...) functionality won't work)
+    """
+
+    flat_lats = dc.lat.data.flatten()
+    flat_lons = dc.lon.data.flatten()
+
+    distances = np.sqrt((flat_lats - lat)**2 + (flat_lons - lon)**2)
```
Comment (mbechtold) [inline help.py:121]: Certainly efficient but increasing error with increasing distance to the equator. Using GDAL's osr and ogr modules, one can create geographic coordinates (points), assign them to a specific coordinate reference system (CRS), and compute the distance.

Comment (mbechtold) [reply help.py:121]: Also fine if you add this as a To Do to your PR and this can be merged for now.

**Review by mbechtold — APPROVED** [review]
Great, thanks for adding those improvements. Only had one comment on the distance calculation to take or leave.

File: help.py (hunk 1 above)
Comment (zdenkoheyvaert) [reply help.py:121]: I flagged this as a TO DO for a future PR!

**Pull request merged**